src/typeahead/typeahead.js

```diff
@@ -499,6 +499,19 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position'])
     };
   }])
 
+  .directive('focusMe', function(){
+     return {
+       restrict: 'A',
+       link: function(scope,element,attrs){
+         scope.$watch(attrs.shouldFocus,function(newValue,oldValue){
```

Comment:
yes! Sorry,my mistake...I mixed up the directive names. Now it's the way it should be